examples/mcp/src/main.tsx

```
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import "./main.css";
+import reactLogo from "./assets/react.svg";
+import FarmLogo from "./assets/logo.png";
+export function Main() {
+  const [count, setCount] = useState(0);
+  console.log("rendering Main component")
+  return (
+    <>
+      <div>
+        <a href="https://farmfe.org/" target="_blank">
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)